config/kafka/config.go

```diff
@@ -233,6 +233,14 @@ func Configure(p *config.Provider) { //nolint:gocyclo
 		}
 		r.MetaResource.ArgumentDocs["secret_arn_list"] = "- (Required) List of all AWS Secrets Manager secret ARNs to associate with the cluster. Secrets not referenced, selected or listed here will be disassociated from the cluster."
 	})
+	p.AddResourceConfigurator("aws_msk_single_scram_secret_association", func(r *config.Resource) {
+		r.References["secret_arn"] = config.Reference{
+			TerraformName:     "aws_secretsmanager_secret",
+			RefFieldName:      "SecretArnRef",
+			SelectorFieldName: "SecretArnSelector",
```

Comment:
Better to leave these unset unless there's a need to change the default name (such as when the generator pluralizes incorrectly)

Reply:
Okay, how can I check if it pluralizes incorrectly?

Reply:
@mbbush https://github.com/crossplane-contrib/provider-upjet-aws/pull/1754/commits/9861ed5f18c967d5eb9bb0ed96c931cca1e7f2dd - with unset reference

Reply:
That's exactly what I had in mind. Thanks!